backend/package.json

```diff
@@ -25,21 +25,26 @@
     "knex": "./scripts/knex.sh",
     "migrate": "./scripts/migrate_latest.sh",
     "migrate:all": "node ./scripts/createSqlDatabases.js && yarn migrate auth",
-    "migrate:make": "./scripts/migrate_make.sh"
+    "migrate:make": "./scripts/migrate_make.sh",
+    "seed:question-difficulties": "node ./scripts/seedQuestionDifficulties.js",
+    "seed:question-categories": "node ./scripts/seedQuestionCategories.js"
```

Comment:
For convenience, is it possible to update the migrate:make command to also help seed the mongodb

Reply:
Done!